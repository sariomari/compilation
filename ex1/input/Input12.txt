586
 1311
/*l);4)g!!*//*]?(*//*!;5}]}!*/6
849
7
8746
736
1
37
/*}
*/8881
/**/140
""/*T]l!!
;
;/a?p*//*()]
y?!F]};
7
!;??;)*///;[}]j?"""h""xmcULS"/**/403
/*}]};?
*/pN63	  /*B
?3*/QS820
"HBSj"7375
6
"GjMb"756
"RT"6
k	/*?.}5
?
?;r?]*/426
12
3
/*c[;)3T!;
?;l})}h??]}

!T)*/102
8
6661
491
"kNX""AOQQnSzY"//(];LR
/*;v;5];!!}Y9!*/jH""u4f0We4Q337
588
319
41
  6
hqgu3678
76
"wqT"/*]?(*/49
88
865
17
//;18
5
H6F91
4243
es1tx5Vzwm3
//4}
mnl7"MRf""X"1
M""cX648
//]
 56
1782
8
37
776
 ////t?
229
707
48
//);;5129
//;8432
2752
756
 /*

)]

y*//*?*///}(!4
2
//;!)5?;];]};5?}
s""//
/*;;v*/Ink65065
67
 /*P!}!]
??;u!]T!)E]
k*/6657
x/*???D1*/664
i//u};Nz?]!;;
//!} "UHKV"35
8739
	//h!])K4//4}"oM" ""4401
/*eo]k87;;?H?sN?/A;?!!5i)
A9Y!!(*/1
950
/*}*/C"JeQC"/*}!2!;q}]
(}*/580
9137
""//S	209
647
p	450
607
2
7
Z0aS976"jAP"/*(/;?F)8e.;*/60
4864
2112
	2
 /*
Tm*//*!L*///]!;
3503
	65
5
574
mPH8927
15
"J"J9328"tAj"4003
//E	FOe4//)
"eaN" 657
p09e//}31
2662
If 92
//;!]4;/*)6;8;J*/ak1rlh8143
8
132
249
 6066
Z/*7]id;J
;v
(
)])]
1}((
?})N!?;]
Ba*/d82	o997
//j7
//1175
5564
 78
512
3
231
/*z!)}!?};!;]!;!
]?(D2r!U!!()}
]!(]]*/6622
5
52
91
244
k8EkE3CY268688
88
"bAO"445
/*
6};9*/3
308
7
3782
//;!?)!"nx"s1242
142
0
 "udjyg"5
//(5w309
653
//]??2?h//I
	/*p*/99
//?
/*2M}2?A}};;
n!;;;!
]Xl1!(b*/ 921
315
//]dvP;?]]
//n?}];q)(26
7474
//0?!?](}12
//;80!!1Ws
1721
6298
0
//P;l
/*jP;!y;]8Z!!(]*/	83
""//!;
zJ6777
4101
//);)}c
6
7573
2229
"ZFkHaCt"915
298
93
AUSUZ7K""//]/Y}W
//637
//(;
7850
324
//}!]]}??h]!;CEixj}N??
9
44
4
 77
3
4974
u/*T4!(};??*/ 8
	269
"bnfz""NQ"24
4640
Py37G5
287
/**///Tl;P]h(];q}(2(l}179
//Di)!R}E(;
2020
3987
GO/*?8;]3)]!;]];0{*/17
//g!?"EaX"23
Vj2/*??
c)J;!}
]!
7Q*//*
;)R(;;]
]
]N;*/ 	//]Z0(
/*T]7;(*/""u//l
3
//}?e/*ZFg5k]}?!};P6
jc?P?!p})??EvA(K;/*///]T;!4339
7
"Zi""RgVJiTvcGaf"f6090
37
aztfwEctM			/*1?;!R);;}R
a(!)8q!I}]?)
z;H;]];6!
?;])
(]j]((?4*///}!566
"y"//85m
9494
6530
1307
hHAdx	cYDA  20
86
216
milm2
""0
717
47
993
7616
rzhNT	//M(;}RO
//}x
274
/*1?S;?!;FZ}v?}*/5
	//
	gGZ964
//]vLs;Z??;?(?!!b!/19
ox5J7d //;?!}wa689
4364
/*
{*/103
771
318
Vs6
//
9234
519
 0
/*
h?
)Q?2??
*/4
5
8
2
/*?;V(]*/309
178
"n"80
2
//]}!}?C/*O*//*(?o]}
?({;!)!;;!]}!*///;;5i0
422
32
8387
//]
/*P7
*/Jd84905
486
M3
//;!(9[;NhK335
238
1413
P3
sN96"wu"	//g89(!288
1044
"C"3
//}J!)
52
8219
LTIe9Aztj8//];;4{""381
9
/**/k/*!?!]}D!?W*/7
  /*}]d
7};)}*/ 5486
//42
//	8552
	 /*W*/216
//)}
/*]3HN)()r
}!H((!]
!}!]w
](*/1
 13
"gbe"3132
2
N87
512
"fMSSy""D"//((]i!3d!NB"z" /*

(;;]*/97
			/*lq?/;;wf]H
V?;5];;(m*/rF 664
702
822
858
40
15
18
306
68
28
83
UHx9704772
54
//!
5
//(zh838
/*!*/3
//40
/*A
!pTp*/"KSAE"817
"f"J"j""""F"hg1
//}37984
//?q6
"jTSVm"1355
4
4
0
2
s9	0
A72k//1};}P4
/**/69
u29994
"qm"/*)E}!*/3
//a35137
2870
//o!}(?}]q??!;!;;K6B43
"x"k"dBbWDTeWMZZcH"325
"c"/*1}036
T
k)]?.5}?b?(
[?
WA}}}?]?;!u/LqZ]?S?;]
)!}*//*
e]kka]8
P8}4*/"R"615
yv307
70
520
//]M!
/*V;
]?;;U]}]3N}S}?0)*/905
//))?;!;	""69
/*g(8
?
2}?*/""""57
342
9
"qLbp"NL4 /*W}!maU?}3}!K!
]*/174
	B9471
//? 561
 "CLZRtI" //]?}
Y/*!?;
f}f*//*/;)sB*/ 8754
721
//;67
"yHDGD"7
64
"Y"6
  22
2471
"LiX""gTz"/*9?!w!]8]})}*//*!6}sU;};?(
?]]
(
s;!?;D/O!
E;?

Y
G]*/D946
14
k5
55
44
43
""9270
""""""9
5
	QxXR480y88
	4
/*j3;h}?N{]9Z
!
*/332
469
	 93
wM"q"x36Y6QDVdi1576
 ""	62
"t"//]h}?;;))3)K
""/**/57
kz9Tf49Iu"wM"90
/*??}h]}2}
(*//*1?)j(l?/;*/1
//?T!K};
""1220
67
	296
	27
//]!;!"Sk""z"A4//i?cx;587
/*}]*/ 13
 9592
/*(
})G?Q}b;??)!]*//*?5;O*/"PT"/*(;(]*/9743
647
"tZ"9
	6360
9
9414
5611
3
9656
/*?*/71
98
34
//1!}
0
//// 89
8
2106
/*?8b?q
!
*/777
	//?;f)WS;??})W
 4020
//;r/*!?4(8?xa4}

![)*///!?)4?))]))]?(!
0
709
/*D}!/]}}!M?w;Z]}]N)6H
?*/4988
"XbYi"""5701
	9058
//()0uz}s
11
797
o422
540
4320
2032
991
	143
5
946
/*?;E(
?;]r}m0!*///Bd(!t6bX5g	453
7
/*}1);;}]*///3103
71
//))1
 	4118
4826
8
U1v"Sz"//Ig(Sl?b 75
94
5
 	//450
938
89
61
U6399
1
/*C!tk(h}3
*/2
"Flbwl" 583
/*(
r;}*//*}}m?]?s
5}t}N]M
F98
*/C81
4
50
"Rlgn"vBM484
655
"HfGgx""VS"	DS9""//0!;}/**/1451
""43
"yNn"	919
0
8
//6l(1749
7
 Z6
7
//;
//]}!C03467
/**/228
0
"DO""Lz"6
	"NINlAq"Jt72
7
265
 81
//}N(?;]](]""3942
66
/*38*/363
431
/*(!
I)Vv!}c!!*/89
55
7646
2
D9R//8I2;[I12919
80
	/*
Iq
H
};};?!2([]??o9V;*//*?}}]?())9ID.*/2
Pw1vx2
	U583
3098
9
4
 157
//!]}
""Z//};/!
7
"pW"82
5
"KW"1
/*
!}?!}
?8?lB;;(/.;
?!];.
5;;*//*ql]iB!!*//**/	7326
"RXcU"/*!
;*/3
zr25507
/*.si!9})!(*///;)((6(j(}].;(}/!{]s!?}}"Yy"5435
4
25
423
bFT6
/*!0]!]?
;)!}
;??sr/*//*W*/7
J9566
//;}"Vr"//(//
""321
//
 6
"WNfk"3
20
"c"7952
//?633
	md/*9]!
;;.U?;!!!}?r?!}*/ ch	o	ff3822
 	f71850
cgwU2626
673
"rs"9641
//]
	/*!!/7!*/yAbpHAln407
""6
"kOwalvAqMLnRo"668
 39
 3
5469
xmU9743
//]!]]!
/*!?*/jk9eUk48
10
F4
2
40
3370
95
4266
34
//;LB33I3v9
"R"2845
/*?}PpOm;I?
!(?(!*/ 9
3922
"vGTI"/*?*/83
910
/*]]rZ/;;
!];}Ro
*/3562
0
//!p]))//}/*)((}?
!P
!3i(;*/