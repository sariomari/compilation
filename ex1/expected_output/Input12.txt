INT(586)[1,1]
INT(1311)[2,2]
INT(6)[6,23]
INT(849)[7,1]
INT(7)[8,1]
INT(8746)[9,1]
INT(736)[10,1]
INT(1)[11,1]
INT(37)[12,1]
INT(8881)[14,3]
INT(140)[15,5]
STRING("")[16,1]
STRING("")[25,1]
STRING("h")[25,3]
STRING("xmcULS")[25,6]
INT(403)[25,18]
ID(pN63)[27,3]
ID(QS820)[28,5]
STRING("HBSj")[29,1]
INT(7375)[29,7]
INT(6)[30,1]
STRING("GjMb")[31,1]
INT(756)[31,7]
STRING("RT")[32,1]
INT(6)[32,5]
ID(k)[33,1]
INT(426)[36,8]
INT(12)[37,1]
INT(3)[38,1]
INT(102)[45,6]
INT(8)[46,1]
INT(6661)[47,1]
INT(491)[48,1]
STRING("kNX")[49,1]
STRING("AOQQnSzY")[49,6]
ID(jH)[52,7]
STRING("")[52,9]
ID(u4f0We4Q337)[52,11]
INT(588)[53,1]
INT(319)[54,1]
INT(41)[55,1]
INT(6)[56,3]
ID(hqgu3678)[57,1]
INT(76)[58,1]
STRING("wqT")[59,1]
INT(49)[59,13]
INT(88)[60,1]
INT(865)[61,1]
INT(17)[62,1]
INT(18)[64,1]
INT(5)[65,1]
ID(H6F91)[66,1]
INT(4243)[67,1]
ID(es1tx5Vzwm3)[68,1]
ID(mnl7)[70,1]
STRING("MRf")[70,5]
STRING("X")[70,10]
INT(1)[70,13]
ID(M)[71,1]
STRING("")[71,2]
ID(cX648)[71,4]
INT(56)[73,2]
INT(1782)[74,1]
INT(8)[75,1]
INT(37)[76,1]
INT(776)[77,1]
INT(229)[80,1]
INT(707)[81,1]
INT(48)[82,1]
INT(129)[84,1]
INT(8432)[86,1]
INT(2752)[87,1]
INT(756)[88,1]
INT(2)[94,1]
ID(s)[96,1]
STRING("")[96,2]
ID(Ink65065)[97,8]
INT(67)[98,1]
INT(6657)[103,4]
ID(x)[104,1]
INT(664)[105,4]
ID(i)[106,1]
STRING("UHKV")[108,2]
INT(35)[108,8]
INT(8739)[109,1]
STRING("oM")[112,1]
STRING("")[112,6]
INT(4401)[112,8]
INT(1)[115,9]
INT(950)[116,1]
ID(C)[117,6]
STRING("JeQC")[117,7]
INT(580)[118,5]
INT(9137)[119,1]
STRING("")[120,1]
INT(209)[121,2]
INT(647)[122,1]
ID(p)[123,1]
INT(450)[123,3]
INT(607)[124,1]
INT(2)[125,1]
INT(7)[126,1]
ID(Z0aS976)[127,1]
STRING("jAP")[127,8]
INT(60)[129,9]
INT(4864)[130,1]
INT(2112)[131,1]
INT(2)[132,2]
INT(3503)[135,1]
INT(65)[136,2]
INT(5)[137,1]
INT(574)[138,1]
ID(mPH8927)[139,1]
INT(15)[140,1]
STRING("J")[141,1]
ID(J9328)[141,4]
STRING("tAj")[141,9]
INT(4003)[141,14]
ID(FOe4)[143,2]
STRING("eaN")[144,1]
INT(657)[144,7]
ID(p09e)[145,1]
INT(31)[146,1]
INT(2662)[147,1]
ID(If)[148,1]
INT(92)[148,4]
ID(ak1rlh8143)[150,11]
INT(8)[151,1]
INT(132)[152,1]
INT(249)[153,1]
INT(6066)[154,2]
ID(Z)[155,1]
ID(d82)[162,5]
ID(o997)[162,9]
INT(1175)[165,1]
INT(5564)[166,1]
INT(78)[167,2]
INT(512)[168,1]
INT(3)[169,1]
INT(231)[170,1]
INT(6622)[174,8]
INT(5)[175,1]
INT(52)[176,1]
INT(91)[177,1]
INT(244)[178,1]
ID(k8EkE3CY268688)[179,1]
INT(88)[180,1]
STRING("bAO")[181,1]
INT(445)[181,6]
INT(3)[183,7]
INT(308)[184,1]
INT(7)[185,1]
INT(3782)[186,1]
STRING("nx")[188,1]
ID(s1242)[188,5]
INT(142)[189,1]
INT(0)[190,1]
STRING("udjyg")[191,2]
INT(5)[191,9]
INT(309)[193,1]
INT(653)[194,1]
INT(99)[197,7]
INT(921)[201,11]
INT(315)[202,1]
INT(26)[205,1]
INT(7474)[206,1]
INT(12)[208,1]
INT(1721)[210,1]
INT(6298)[211,1]
INT(0)[212,1]
INT(83)[216,8]
STRING("")[217,1]
ID(zJ6777)[218,1]
INT(4101)[219,1]
INT(6)[221,1]
INT(7573)[222,1]
INT(2229)[223,1]
STRING("ZFkHaCt")[224,1]
INT(915)[224,10]
INT(298)[225,1]
INT(93)[226,1]
ID(AUSUZ7K)[227,1]
STRING("")[227,8]
INT(637)[229,1]
INT(7850)[231,1]
INT(324)[232,1]
INT(9)[234,1]
INT(44)[235,1]
INT(4)[236,1]
INT(77)[237,2]
INT(3)[238,1]
INT(4974)[239,1]
ID(u)[240,1]
INT(8)[241,9]
INT(269)[242,2]
STRING("bnfz")[243,1]
STRING("NQ")[243,7]
INT(24)[243,11]
INT(4640)[244,1]
ID(Py37G5)[245,1]
INT(287)[246,1]
INT(179)[248,1]
INT(2020)[250,1]
INT(3987)[251,1]
ID(GO)[252,1]
INT(17)[253,12]
STRING("EaX")[255,1]
INT(23)[255,6]
ID(Vj2)[256,1]
STRING("")[268,4]
ID(u)[268,6]
INT(3)[269,1]
INT(4339)[278,1]
INT(7)[279,1]
STRING("Zi")[280,1]
STRING("RgVJiTvcGaf")[280,5]
ID(f6090)[280,18]
INT(37)[281,1]
ID(aztfwEctM)[282,1]
INT(566)[294,1]
STRING("y")[295,1]
INT(9494)[296,1]
INT(6530)[297,1]
INT(1307)[298,1]
ID(hHAdx)[299,1]
ID(cYDA)[299,7]
INT(20)[299,13]
INT(86)[300,1]
INT(216)[301,1]
ID(milm2)[302,1]
STRING("")[303,1]
INT(0)[303,3]
INT(717)[304,1]
INT(47)[305,1]
INT(993)[306,1]
INT(7616)[307,1]
ID(rzhNT)[308,1]
INT(274)[310,1]
INT(5)[313,4]
ID(gGZ964)[315,2]
INT(19)[317,1]
ID(ox5J7d)[318,1]
INT(689)[319,1]
INT(4364)[320,1]
INT(103)[322,4]
INT(771)[323,1]
INT(318)[324,1]
ID(Vs6)[325,1]
INT(9234)[327,1]
INT(519)[328,1]
INT(0)[329,2]
INT(4)[334,3]
INT(5)[335,1]
INT(8)[336,1]
INT(2)[337,1]
INT(309)[338,10]
INT(178)[339,1]
STRING("n")[340,1]
INT(80)[340,4]
INT(2)[341,1]
INT(422)[347,1]
INT(32)[348,1]
INT(8387)[349,1]
ID(Jd84905)[353,3]
INT(486)[354,1]
ID(M3)[355,1]
ID(NhK335)[357,1]
INT(238)[358,1]
INT(1413)[359,1]
ID(P3)[360,1]
ID(sN96)[361,1]
STRING("wu")[361,5]
INT(288)[362,1]
INT(1044)[363,1]
STRING("C")[364,1]
INT(3)[364,4]
INT(52)[366,1]
INT(8219)[367,1]
ID(LTIe9Aztj8)[368,1]
STRING("")[369,1]
INT(381)[369,3]
INT(9)[370,1]
ID(k)[371,5]
INT(7)[371,19]
INT(5486)[373,9]
INT(42)[375,1]
INT(8552)[377,2]
INT(216)[378,8]
INT(1)[388,5]
INT(13)[389,2]
STRING("gbe")[390,1]
INT(3132)[390,6]
INT(2)[391,1]
ID(N87)[392,1]
INT(512)[393,1]
STRING("fMSSy")[394,1]
STRING("D")[394,8]
ID(NB)[395,1]
STRING("z")[395,3]
INT(97)[397,7]
ID(rF)[401,12]
INT(664)[401,15]
INT(702)[402,1]
INT(822)[403,1]
INT(858)[404,1]
INT(40)[405,1]
INT(15)[406,1]
INT(18)[407,1]
INT(306)[408,1]
INT(68)[409,1]
INT(28)[410,1]
INT(83)[411,1]
ID(UHx9704772)[412,1]
INT(54)[413,1]
INT(5)[415,1]
INT(838)[417,1]
INT(3)[418,6]
INT(40)[420,1]
STRING("KSAE")[423,6]
INT(817)[423,12]
STRING("f")[424,1]
ID(J)[424,4]
STRING("j")[424,5]
STRING("")[424,8]
STRING("F")[424,10]
ID(hg1)[424,13]
INT(7984)[426,1]
INT(6)[428,1]
STRING("jTSVm")[429,1]
INT(1355)[429,8]
INT(4)[430,1]
INT(4)[431,1]
INT(0)[432,1]
INT(2)[433,1]
ID(s9)[434,1]
INT(0)[434,4]
ID(A72k)[435,1]
ID(P4)[436,1]
INT(69)[437,5]
ID(u29994)[438,1]
STRING("qm")[439,1]
INT(3)[439,13]
ID(a35137)[441,1]
INT(2870)[442,1]
ID(K6B43)[444,1]
STRING("x")[445,1]
ID(k)[445,4]
STRING("dBbWDTeWMZZcH")[445,5]
INT(325)[445,20]
STRING("c")[446,1]
STRING("R")[463,4]
INT(615)[463,7]
ID(yv307)[464,1]
INT(70)[465,1]
INT(520)[466,1]
INT(905)[471,9]
STRING("")[473,2]
INT(69)[473,4]
STRING("")[476,6]
STRING("")[476,8]
INT(57)[476,10]
INT(342)[477,1]
INT(9)[478,1]
STRING("qLbp")[479,1]
ID(NL4)[479,7]
INT(174)[480,4]
ID(B9471)[481,2]
INT(561)[483,2]
STRING("CLZRtI")[484,2]
ID(Y)[485,1]
INT(8754)[486,16]
INT(721)[487,1]
INT(7)[489,1]
STRING("yHDGD")[490,1]
INT(7)[490,8]
INT(64)[491,1]
STRING("Y")[492,1]
INT(6)[492,4]
INT(22)[493,3]
INT(2471)[494,1]
STRING("LiX")[495,1]
STRING("gTz")[495,6]
ID(D946)[504,4]
INT(14)[505,1]
ID(k5)[506,1]
INT(55)[507,1]
INT(44)[508,1]
INT(43)[509,1]
STRING("")[510,1]
INT(9270)[510,3]
STRING("")[511,1]
STRING("")[511,3]
STRING("")[511,5]
INT(9)[511,7]
INT(5)[512,1]
ID(QxXR480y88)[513,2]
INT(4)[514,2]
INT(332)[517,3]
INT(469)[518,1]
INT(93)[519,3]
ID(wM)[520,1]
STRING("q")[520,3]
ID(x36Y6QDVdi1576)[520,6]
STRING("")[521,2]
INT(62)[521,5]
STRING("t")[522,1]
STRING("")[523,1]
INT(57)[523,7]
ID(kz9Tf49Iu)[524,1]
STRING("wM")[524,10]
INT(90)[524,14]
INT(1)[527,17]
STRING("")[529,1]
INT(1220)[529,3]
INT(67)[530,1]
INT(296)[531,2]
INT(27)[532,2]
STRING("Sk")[534,1]
STRING("z")[534,5]
ID(A4)[534,8]
INT(587)[535,1]
INT(13)[537,4]
INT(9592)[538,2]
STRING("PT")[541,12]
INT(9743)[541,24]
INT(647)[542,1]
STRING("tZ")[543,1]
INT(9)[543,5]
INT(6360)[544,2]
INT(9)[545,1]
INT(9414)[546,1]
INT(5611)[547,1]
INT(3)[548,1]
INT(9656)[549,1]
INT(71)[550,6]
INT(98)[551,1]
INT(34)[552,1]
INT(0)[554,1]
INT(89)[557,2]
INT(8)[558,1]
INT(2106)[559,1]
INT(777)[563,3]
INT(4020)[565,2]
INT(0)[572,1]
INT(709)[573,1]
INT(4988)[575,4]
STRING("XbYi")[576,1]
STRING("")[576,7]
INT(5701)[576,9]
INT(9058)[577,2]
INT(11)[579,1]
INT(797)[580,1]
ID(o422)[581,1]
INT(540)[582,1]
INT(4320)[583,1]
INT(2032)[584,1]
INT(991)[585,1]
INT(143)[586,2]
INT(5)[587,1]
INT(946)[588,1]
ID(bX5g)[592,1]
INT(453)[592,6]
INT(7)[593,1]
INT(3103)[595,1]
INT(71)[596,1]
INT(1)[598,1]
INT(4118)[599,3]
INT(4826)[600,1]
INT(8)[601,1]
ID(U1v)[602,1]
STRING("Sz")[602,4]
INT(75)[603,2]
INT(94)[604,1]
INT(5)[605,1]
INT(450)[607,1]
INT(938)[608,1]
INT(89)[609,1]
INT(61)[610,1]
ID(U6399)[611,1]
INT(1)[612,1]
INT(2)[615,3]
STRING("Flbwl")[616,1]
INT(583)[616,9]
ID(C81)[624,3]
INT(4)[625,1]
INT(50)[626,1]
STRING("Rlgn")[627,1]
ID(vBM484)[627,7]
INT(655)[628,1]
STRING("HfGgx")[629,1]
STRING("VS")[629,8]
ID(DS9)[629,13]
STRING("")[629,16]
INT(1451)[630,5]
STRING("")[631,1]
INT(43)[631,3]
STRING("yNn")[632,1]
INT(919)[632,7]
INT(0)[633,1]
INT(8)[634,1]
INT(1749)[636,1]
INT(7)[637,1]
ID(Z6)[638,2]
INT(7)[639,1]
ID(C03467)[642,1]
INT(228)[643,5]
INT(0)[644,1]
STRING("DO")[645,1]
STRING("Lz")[645,5]
INT(6)[645,9]
STRING("NINlAq")[646,2]
ID(Jt72)[646,10]
INT(7)[647,1]
INT(265)[648,1]
INT(81)[649,2]
STRING("")[651,1]
INT(3942)[651,3]
INT(66)[652,1]
INT(363)[653,7]
INT(431)[654,1]
INT(89)[657,5]
INT(55)[658,1]
INT(7646)[659,1]
INT(2)[660,1]
ID(D9R)[661,1]
ID(I12919)[662,1]
INT(80)[663,1]
INT(2)[669,26]
ID(Pw1vx2)[670,1]
ID(U583)[671,2]
INT(3098)[672,1]
INT(9)[673,1]
INT(4)[674,1]
INT(157)[675,2]
STRING("")[677,1]
ID(Z)[677,3]
INT(7)[678,1]
STRING("pW")[679,1]
INT(82)[679,5]
INT(5)[680,1]
STRING("KW")[681,1]
INT(1)[681,5]
INT(7326)[687,13]
STRING("RXcU")[688,1]
INT(3)[689,4]
ID(zr25507)[690,1]
STRING("Yy")[694,1]
INT(5435)[694,5]
INT(4)[695,1]
INT(25)[696,1]
INT(423)[697,1]
ID(bFT6)[698,1]
INT(7)[705,14]
ID(J9566)[706,1]
STRING("Vr")[708,1]
STRING("")[710,1]
INT(321)[710,3]
INT(6)[712,2]
STRING("WNfk")[713,1]
INT(3)[713,7]
INT(20)[714,1]
STRING("c")[715,1]
INT(7952)[715,4]
INT(633)[717,1]
ID(md)[718,2]
ID(ch)[719,19]
ID(o)[719,22]
ID(ff3822)[719,24]
ID(f71850)[720,3]
ID(cgwU2626)[721,1]
INT(673)[722,1]
STRING("rs")[723,1]
INT(9641)[723,5]
ID(yAbpHAln407)[727,4]
STRING("")[728,1]
INT(6)[728,3]
STRING("kOwalvAqMLnRo")[729,1]
INT(668)[729,16]
INT(39)[730,2]
INT(3)[731,2]
INT(5469)[732,1]
ID(xmU9743)[733,1]
ID(jk9eUk48)[735,7]
INT(10)[736,1]
ID(F4)[737,1]
INT(2)[738,1]
INT(40)[739,1]
INT(3370)[740,1]
INT(95)[741,1]
INT(4266)[742,1]
INT(34)[743,1]
ID(B33I3v9)[745,1]
STRING("R")[746,1]
INT(2845)[746,4]
INT(9)[749,8]
INT(3922)[750,1]
STRING("vGTI")[751,1]
INT(83)[751,12]
INT(910)[752,1]
INT(3562)[756,3]
INT(0)[757,1]